README.md

```diff
@@ -0,0 +1,64 @@
+# cloud-provider-aws
+The AWS cloud provider provides the interface between a Kubernetes cluster and AWS service APIs. This project allows a Kubernetes cluster to provision, monitor and remove resources necessary for operation of the cluster.
+
+## AWS Labels
+Before building out your cluster, you will need to make sure the your AWS resources are properly labeled. Label all k8s instances, your security group, vpc, and subnets with:
+
+KubernetesCluster=<your_cluster_name>
+
+## Flags
+The flag --cloud-provider=aws needs to be passed to kubelet, apiserver, and controller-manager.
+
+## IAM Policy
+For your Kubernetes cluster to be able to create and modify AWS resources, you will need to create a few IAM policies for your EC2 instances. The master policy is a bit open and can be scaled back depending on the use case. Adjust these based on your needs.
+### 1. Master Policy
+
+  {  
+  
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+              "Effect": "Allow",
+              "Action": [
+                  "ec2:*",
+                  "elasticloadbalancing:*",
+                  "ecr:GetAuthorizationToken",
+                  "ecr:BatchCheckLayerAvailability",
+                  "ecr:GetDownloadUrlForLayer",
+                  "ecr:GetRepositoryPolicy",
+                  "ecr:DescribeRepositories",
+                  "ecr:ListImages",
+                  "ecr:BatchGetImage"    
+              ],
+              "Resource": "*"   
+          }     
+      ]
+  }
+
+### 2. Node Policy
+  {
```

Comment:
Identation needs fixing here too (or use backticks instead).  Also, add a newline in between the title and code block.